src/main/java/step2/domain/LottoMachine.java

```diff
@@ -0,0 +1,43 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoMachine {
+    private final static int LOTTO_PRICE = 1_000;
```

Comment:
```
The Java Language Specification recommends listing modifiers in the following order:
1. Annotations
2. public
3. protected
4. private
5. abstract
6. static
7. final
8. transient
9. volatile
10. synchronized
11. native
12. strictfp
```

Reply:
위에 명시된 순서대로 해당 코드를 수정하면 아래와 같습니다.
```suggestion
    private static final int LOTTO_PRICE = 1_000;
```